dependencyManagement/build.gradle.kts

```diff
@@ -11,7 +11,9 @@ data class DependencySet(val group: String, val version: String, val modules: Li
 val dependencyVersions = hashMapOf<String, String>()
 rootProject.extra["versions"] = dependencyVersions
 
+// IMPORTANT when updating otelVersion, need to also update grpcVersion
```

Comment:
The main requirement is alignment - so actually if grpc-bom were to work (it can definitely be made to work but I'm not sure it currently does without some tweaks in grpc modules, see other comment) then the versions of gRPC and otel become decoupled in our build and gRPC versions would be aligned, possibly on a different version than specified by OTel. The policy would then be always use the latest version of dependencies when releasing, i.e., there wouldn't be anything intrinsically different about grpc vs jackson.